vmm/src/device_manager/mmio.rs

```diff
@@ -367,6 +367,18 @@ mod tests {
             let _ = value;
         }
 
+        fn get_available_features(&self) -> u64 {
+            0
+        }
+
+        fn get_acked_features(&self) -> u64 {
+            0
+        }
+
+        fn set_acked_features(&mut self, acked_features: u64) {
```

Comment:
I would do it this way, it's less code:
```suggestion
        fn set_acked_features(&mut self, _acked_features: u64) {}
```
or 
```suggestion
        fn set_acked_features(&mut self, _: u64) {}
```